Elfie/SymbolSourceGet/App.config

```diff
@@ -1,17 +1,29 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6"/>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6" />
   </startup>
   <system.diagnostics>
     <trace autoflush="true" indentsize="4">
       <listeners>
-        <add name="consoleListener" type="System.Diagnostics.ConsoleTraceListener"/>
+        <add name="consoleListener" type="System.Diagnostics.ConsoleTraceListener" />
       </listeners>
     </trace>
   </system.diagnostics>
   <appSettings>
-    <add key="SymbolServerUrls" value="http://referencesource.microsoft.com/symbols|http://msdl.microsoft.com/download/symbols|https://nuget.smbsrc.net"/>
-    <add key="SymbolCachePath" value="%SystemDrive%\SymbolCache"/>
+    <add key="SymbolServerUrls" value="http://referencesource.microsoft.com/symbols|http://msdl.microsoft.com/download/symbols|https://nuget.smbsrc.net" />
+    <add key="SymbolCachePath" value="%SystemDrive%\SymbolCache" />
   </appSettings>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
It was added automatically by the "Manage NuGet References" tool, so I think so (I suspect the currently referenced Roslyn references an older version of System.Reflection.Metadata.)